.reuse/dep5

```diff
@@ -6,20 +6,15 @@ Source: https://github.com/latchset/pkcs11-provider/
 #
 # Build system, data files from tests, and misc cruft
 #
-Files: **/Makefile.am
-       .github/*
+Files: .github/*
        .gitignore
-       Makefile.am
+       GNUmakefile
```

Comment:
Why GNUmakefile instead of the common Makefile?

Reply:
There are a couple of minor motivations here. First, you wouldn't need to care about compatibility with other make implementations, as GNUmakefile is only recognized by GNU make. Secondly, with GNU make, you could write the current rules slightly simpler, using wildcards without shell functions (I included this change in https://github.com/latchset/pkcs11-provider/pull/304/commits/30c77f5acda9f11c143a4c37504ddb35809eb195 but can revert it if you don't like it).

Reply:
I think we are making it hard enough for other platforms already with the switch to meson, let's just use a standard Makefile please.

Reply:
Reverted.